node 16 debt: remove ./test/EditLog export (#23244)
microsoft/FluidFramework: .changeset/gentle-needles-own.md

---

> "@fluid-private/test-end-to-end-tests": minor

Reviewer: I don't think there is any reason to include a changeset for anything in @fluid-private: it should not have any users and thus does not need a change log entry informing users about this change.
```suggestion
```

Author: Updated with suggested change

"@fluid-experimental/tree": minor
---
---

> "section": legacy

Reviewer 2: Why legacy?

Author: Updated to `deprecated`, but wasn't sure if this is correct. Would `other` be more appropriate since we are not explicitly adding deprecated tags?

> "section": deprecation

Reviewer: Generally, a deprecation does not remove/move an API, but instead just discourages its use ( see https://en.wikipedia.org/wiki/Deprecation ). This is a breaking change moving an API, so I don't think "deprecation" is the right section. I don't have an easy way to get a list of what the options are other than generating new changeset to see what it allows, but I think there is a more appropriate section (even if its just other)

Author: The other choices are "other" and "legacy API changes" when generating the changesets. I updated to "other" for now, but would "legacy API changes" be more appropriate?

Reviewer: I think "other" is correct, as the APIs in question are not tagged with `@legacy`. Perhaps it would have made more sense for those APIs to be legacy instead of experimental, but that's a different issue.

---

Adds `getLocalEdits` method to OrderedEditSet interface and removed `./test/EditLog` export

> The places that imported `EditLog` were to cast to this type  for access to the `getLocalEdits` method. This change removes the need for this cast, so we can get rid of the `./test/EditLog` export.

Reviewer 2: typo: extra space in "type  for".

Author: Fixed typo

> The places that imported `EditLog` were to cast to this type for access to the `getLocalEdits` method. This change removes the need for this cast, so we can get rid of the `./test/EditLog` export.

Reviewer 3: ```suggestion
Previously, `EditLog` was imported to cast to this type for access to the `getLocalEdits` method. This change removes the need for this cast, so we can get rid of the `./test/EditLog` export.
```
